sdk/storage/azure-storage-common/inc/azure/storage/common/crypt.hpp

```diff
@@ -7,32 +7,21 @@
 #include <string>
 #include <vector>
 
+#include "azure/core/cryptography/hash.hpp"
 #include <azure/core/base64.hpp>
 
 namespace Azure { namespace Storage {
```

Comment:
@JeffreyRichter, @RickWinter should this be in a Cryptography sub-namespace, i.e. `Azure::Storage::Cryptography`?

Reply:
Yes, I suggested Crypto b/c that is what some other languages use.  I'm open to any good name.  I suspect the set grows and thus my push for them to be in a namespace.

Reply:
Right, @RickWinter, that makes sense for `Azure::Core::Cryptography` and the base `Hash` class (which we already did). Does that apply to services that derive from the base as well, such as Crc64 in Storage?